wiki/Client/Menu_tips/de.md

```diff
@@ -92,24 +90,36 @@ Diese Tipps erscheinen im [Beatmap-Editor](/wiki/Client/Beatmap_editor) unter de
 
 ## osu!(lazer)
 
-Diese Tipps erscheinen beim Start von [osu!(lazer)](/wiki/Client/Release_stream/Lazer), vor der Intro-Sequenz. Für diesen Artikel wurden die Tipps übersetzt, da sie im Spiel nur auf Englisch verfügbar sind.
+Diese Tipps erscheinen unten im Hauptmenü von [osu!(lazer)](/wiki/Client/Release_stream/Lazer). Für diesen Artikel wurden die Tipps übersetzt, da sie im Spiel nur auf Englisch verfügbar sind.
 
-- Du kannst überall `Strg` + `T` drücken, um die Toolbar zu aktivieren oder zu deaktivieren!
+- Du kannst überall `Strg` + `T` drücken, um die Symbolleiste zu aktivieren oder zu deaktivieren!
 - Du kannst überall `Strg` + `O` drücken, um auf die Optionen zuzugreifen!
-- Alle Einstellungen sind dynamisch und werden in Echtzeit wirksam. Probiere, das Spiel zu pausieren und den Skin zu ändern!
+- Alle Einstellungen sind dynamisch und werden in Echtzeit wirksam. Probiere, den Skin während des Spiels zu ändern!
 - Mit jedem Update kommen neue Features hinzu. Bleibe immer auf dem Laufenden!
 - Wenn dir die Bedienelemente zu groß oder zu klein sind, versuche, die UI-Skalierung in den Einstellungen anzupassen!
 - Versuche, die `Bildschirmskalierung` anzupassen, um den Bereich der Benutzeroberfläche oder des Gameplays zu ändern, sogar im Vollbildmodus!
-- Was früher "[osu!direct](/wiki/osu!supporter#osu!direct)" war, ist nun wie auf der Webseite für alle Nutzer verfügbar. Du kannst es überall mit `Strg` + `D` aufrufen!
-- In Replays kann durch das Ziehen des Cursors auf der Schwierigkeitsleiste am unteren Rand des Bildschirms gesprungen werden!
+- Was früher "[osu!direct](/wiki/osu!supporter#osu!direct)" war, ist nun wie auf der Webseite für alle Nutzer verfügbar. Du kannst es überall mit `Strg` + `B` aufrufen!
+- In Replays kann durch das Ziehen des Cursors oder durch die linke und rechte Pfeiltaste auf der Fortschrittsleiste am unteren Rand des Bildschirms gesprungen werden!
```

Comment:
The sentence sounds weird after the addition regarding arrow keys. How about

```suggestion
- Du kannst durch Replays springen, indem du die linke oder rechte Pfeiltaste drückst oder den Cursor über die Fortschrittsleiste unten ziehst!
```

Reply:
One `unten` too many.

Reply:
Sure, that was just a suggestion. Feel free to do keep the original or do something else if it sounds better to you.

Reply:
On rereading, I get what this `unten` does there, sorry. I like your suggestion more but the `unten` still seems weird to me. Now I would either leave it out entirely or add this part:

```suggestion
- Du kannst durch Replays springen, indem du die linke oder rechte Pfeiltaste drückst oder den Cursor über die Fortschrittsleiste am unteren Bildschirmrand ziehst!
```